src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_create_with_auto_upgrade_channel.yaml

```diff
@@ -346,27 +467,27 @@ interactions:
       Connection:
       - keep-alive
       ParameterSetName:
-      - --resource-group --name --location --generate-ssh-keys --enable-managed-identity
-        --auto-upgrade-channel
+      - --resource-group --name --location --enable-managed-identity --auto-upgrade-channel
+        --aks-custom-headers --ssh-key-value
```

Comment:
is aks-custom-headers expected here?

Reply:
Yes, this is expected.

I have left a comment in the test case, the content specified by the custom header is deprecated and we are only testing the option.

Reply:
In commit [#ed6b268](https://github.com/Azure/azure-cli/pull/20463/commits/ed6b26865b11729854154551ceb580f2cf5eddb3), I added a new test case to test the `--aks-custom-headers` option and at the same time this option used in other cases has been removed to ensure consistent testing and user behavior.